Project.toml

```diff
@@ -4,6 +4,7 @@ authors = ["Mosè Giordano", "Elliot Saba"]
 version = "1.3.0"
 
 [deps]
+Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
```

Comment:
This should hopefully not affect much loading of the package, right?

Reply:
Correct.